template.txt

```diff
@@ -15,7 +15,7 @@ stanford_path_to_models_jar = "./stanford-corenlp-full-2018-10-05/stanford-coren
 javahome = '*your_path_to_jdk_bin_java.exe*'
 
 #DOCKER variables
-    DOCKER = "Y"
+    DOCKER = "N"
```

Comment:
Changes still not available in `config.py`

Reply:
please check again.. I made the changes